test/images/nettest/nettest.go

```diff
@@ -241,11 +241,21 @@ func contactOthers(state *State) {
 	if err != nil {
 		log.Fatalf("Unable to create client; error: %v\n", err)
 	}
-	// Double check that worked by getting the server version.
-	if v, err := client.Discovery().ServerVersion(); err != nil {
-		log.Fatalf("Unable to get server version: %v\n", err)
-	} else {
-		log.Printf("Server version: %#v\n", v)
+
+	// the pod might not have network connectivity on the first try.
```

Comment:
This comment doesn't actually mean anything?

Better wording might be: `Try to get the server version until <timeout>; we use a timeout because the pod might not have immediate network connectivity.` or something along those lines.

Reply:
Done